maven-scm-providers/maven-scm-provider-bazaar/src/main/java/org/apache/maven/scm/provider/bazaar/BazaarScmProvider.java

```diff
@@ -81,7 +81,12 @@ public ScmProviderRepository makeProviderScmRepository( String scmSpecificUrl, c
     public ScmProviderRepository makeProviderScmRepository( File path )
         throws ScmRepositoryException, UnknownRepositoryStructure
     {
-        if ( path == null || !path.isDirectory() )
+        if ( path == null )
+        {
+            throw new NullPointerException( "Path argument is null" );
```

Comment:
Please say "path cannot be null".

Reply:
I agree but please consider that other ScmProviders use "Path argument is null" (e.g. AbstractSvnScmProvider)

Reply:
Agreed, consistency is important. Leave as-is.